### PR TITLE
Add MACOS install instructions

### PR DESCRIPTION
**Pull request by ahcorde**
Signed-off-by: Alejandro Hernández <ahcorde@gmail.com>

# 🦟 Bug fix

## Checklist
- [x] Signed all commits for DCO
- [ ] Added tests
- [x] Updated documentation (as needed)
- [ ] Updated migration guide (as needed)
- [x] `codecheck` passed (See [contributing](https://ignitionrobotics.org/docs/all/contributing#contributing-code))
- [x] All tests passed (See [test coverage](https://ignitionrobotics.org/docs/all/contributing#test-coverage))
- [x] While waiting for a review on your PR, please help review [another open pull request](https://github.com/pulls?q=is%3Aopen+is%3Apr+user%3Aignitionrobotics+repo%3Aosrf%2Fsdformat+archived%3Afalse+) to support the maintainers

**Note to maintainers**: Remember to use **Squash-Merge**

### CODE REVIEW

**Comment by scpeters** [comment]
there is an easier way; we can do `brew install --only-dependencies ignition-math6` and it will pull the dependencies from the homebrew formula

this is what we do in CI: https://github.com/ignition-tooling/release-tools/blob/master/jenkins-scripts/lib/project-default-devel-homebrew-amd64.bash#L72

File: tutorials/installation.md [hunk 1]
```diff
@@ -32,6 +32,22 @@ sudo apt install libignition-math<#>-dev
...
 
+### macOS
+
+On macOS, add OSRF packages:
+  ```
+  ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+  brew tap osrf/simulation
+  ```
+
+Install Ignition GUI:
```
Comment (scpeters) [inline tutorials/installation.md:43]: GUI -> Math

File: tutorials/installation.md [hunk 2]
```diff
@@ -126,6 +142,34 @@ The optional Eigen component of Ignition Math requires:
...
+1. Clone the repository
+  ```
+  git clone https://github.com/ignitionrobotics/ign-math -b ign-math<#>
+  ```
+  Be sure to replace `<#>` with a number value, such as 1 or 2, depending on
+  which version you need.
+
+2. Install dependencies
+  ```
+  brew install $(sort -u $(find .  -iname 'packages.brew') | tr '\n' ' ')
```
Comment (scpeters) [inline tutorials/installation.md:156]: `brew install --only-dependencies ignition-math<#>`

**Comment by ahcorde** [comment]
Thank your for the tips @scpeters !

**Review by scpeters — APPROVED** [review]

**Pull request merged**